src/components/Dashboard/AddressListSm.vue

```diff
@@ -160,9 +160,9 @@
         </v-expansion-panel-content>
       </v-expansion-panel>
 
-      <!-- Records Office (BEN/BC/CCC/ULC and CORPs) -->
+      <!-- Records Office (BEN/BC/CC/ULC and CBEN/C/CCC/CUL) -->
       <v-expansion-panel
-        v-if="isBaseCompany || isCorporation"
```

Comment:
Corporations (legal type CR) aren't a thing in LEAR or COLIN. This is a Names type, which we map to BC on our side. Similarly, BC in Names maps to BEN on our side.

I checked the Dev, Test and Prod dbs (businesses table) and did not find any with type CR, so I deleted this obsolete getter and its references.